solidity/contracts/LPRewards.sol

```diff
@@ -260,3 +260,13 @@ contract LPRewardsTBTCSaddle is LPRewards {
         gated = _gated;
     }
 }
+
+/// @title KEEP rewards for the tBTC V2 Saddle liquidity pool.
+/// @dev LP token: https://etherscan.io/address/0x122Eca07139EB368245A29FB702c9ff11E9693B7
+///      Pool: https://saddle.exchange/#/pools/tbtc/deposit
+contract LPRewardsTBTCv2Saddle is LPRewards, LPRewardsTBTCSaddle {
```

Comment:
Does it have to extend `LPRewards` given that `LPRewardsTBTCSaddle` already extends it?

Reply:
Right, we don't need to extend it twice c8d520d